simulator/infra/ports/t/unit_test.cpp

```diff
@@ -219,3 +220,154 @@ TEST_CASE("Module: exclusion")
     CHECK_FALSE( h.e->check_if_dumps());
     CHECK( h.f->check_if_dumps());
 }
+
+struct SomeTopology : public BaseTestRoot
```

Comment:
I feel that has to be moved to new translation unit.

Reply:
I added new file to CMakeLists.txt, but still getting include problems. How should I properly add it?

Reply:
What are the problems, could you please send output?

Reply:
And CMakeLists.txt diff please

Reply:
It just doesn't see declarations from module.h
`../infra/ports/t/topology_test.cpp:13:30: error: expected class name
struct SomeTopology : public BaseTestRoot`
https://github.com/exucutional/mipt-mips/blob/be8e5e781c352cebd4a22f37cfdc6e210229dd56/simulator/CMakeLists.txt#L71-L75
https://github.com/exucutional/mipt-mips/blob/be8e5e781c352cebd4a22f37cfdc6e210229dd56/simulator/infra/ports/t/topology_test.cpp#L6-L13

Reply:
See comment below